### ISSUE TITLE
질문) 배치정규화

### ISSUE THREAD

**Issue by seohaa** [lovedlim/tensorflow#27]
중간 레이어에 배치정규화를 시키면 첫번째 레이어에서 나온 값이 바뀌어서 들어간다는 건데 그게 왜 가능한건지..모르겠습니다.

**Comment by teddylee777** [comment]
다음과 같이 layer가 구성되어 있다고 가정해 봤을때

- layer1
- Batch Normalization
- layer2
- Batch Normalization
- layer3

배치정규화(Batch Normalization)는 데이터가 layer1 -> layer2로 넘어가는 단계, 그리고 layer2 -> layer3로 넘어가는 단계에서
배치 단위의 데이터에 대해 정규화를 해줍니다.

layer1 -> layer2로 넘어가는 배치 단위의 데이터가 정규화가 안되어 있을 수 있지만,
배치 정규화를 해주면 정규화된 데이터가 layer2로 들어가게 됩니다.

배치 정규화를 통해 gradient의 소실/폭주를 방지해 주어 좀 더 안정적인 학습이 될 수 있도록 기대할 수 있습니다.

감사합니다.
